client/src/index.ts

```diff
@@ -0,0 +1,48 @@
+const notesGrid: HTMLElement|null = document.querySelector('tbody');
+
+getNotes();
+
+async function getNotes() { 
```

Comment:
What the point to make this function async, and wait the result from `fetch` if you just call it as a global?